app/src/ui/tutorial/tutorial-panel.tsx

```diff
@@ -281,6 +295,10 @@ export class TutorialPanel extends React.Component<
   public onStepSummaryClick = (id: ValidTutorialStep) => {
     this.setState({ currentlyOpenSectionId: id })
   }
+
+  private onPreferencesClick = () => {
+    this.props.dispatcher.showPopup({ type: PopupType.Preferences })
```

Comment:
i think we should pass an `intialTab: PreferencesTab.Advanced` here too so that it opens to the preferences tab where the editor can be configured.

Reply:
I think there may be a small typo in your suggestion @outofambit, but it makes sense to me to pop them to the right tab if we can do that reliably. `initialTab: PreferencesTab.Advanced`

Reply:
hello i am a new developer here in this sight add up in my gmail hangout messenger raymondmortimer43@gmail.com